packages/cli/src/steps/compile/runWebpackCompiler.ts

```diff
@@ -31,18 +31,21 @@ const runWebpackCompiler = ({
 
         const modules = componentModules.map(addDependencies(dependencyPackages));
 
-        const components = Object.keys(entrypoints);
-        const componentsContent = [...components, 'commons'].reduce((contents, componentName) => {
-          const content = compiler.outputFileSystem
-            .readFileSync(`${process.cwd()}/bojagi/${componentName}.js`)
-            .toString();
-          // eslint-disable-next-line no-param-reassign
-          contents[componentName] = content;
+        const entrypointNames = Object.keys(entrypoints);
+
+        const outputContent = [...entrypointNames, 'commons'].reduce((contents, fileName) => {
+          const filePath = `${process.cwd()}/bojagi/${fileName}.js`;
```

Comment:
i guess this is always just mem file system? still should we not match the mem fs structure and the real one? so sth like `.bojagi/<namespace>/<fileName>.js` instead?

Reply:
Yeah, we should match it, so it's possible to also put this to the real FS